client.js

```diff
@@ -224,10 +228,15 @@ function main() {
         summary();
         break;
       case "crawl":
-        let crawler = new ChordCrawler(target.ip, target.port, 3000);
+        let crawler = new ChordCrawler(
+          target.host,
+          target.port,
+          CRAWLER_INTERVAL_MS
+        );
         const express = require("express");
         const app = express();
-        const port = args.webPort || 3000;
+        // TBD is this OR a bug?
+        const port = args.webPort || CRAWLER_INTERVAL_MS;
```

Comment:
This is a bug. They were just using the same magic number. This is the default port that the Express web service listens on.

Reply:
Understood.  Replaced instead with "DEFAULT_HOST_PORT".